content/en/boilerplates/gateway-api-future.md

```diff
@@ -1,4 +1,4 @@
 ---
 ---
-Istio includes beta support for the Kubernetes [Gateway API](https://gateway-api.sigs.k8s.io/) and intends
-to make it the default API for traffic management [in the future](/blog/2022/gateway-api-beta/).
+Istio includes stable support for the Kubernetes [Gateway API](https://gateway-api.sigs.k8s.io/)
+and intends to make it the default API for traffic management [in the future](/blog/2024/gateway-mesh-ga/).
```

Comment:
Since the reference blog doesn't mention default API or the future, maybe change the text to something like this?
```suggestion
Istio includes stable support for the Kubernetes [Gateway API](https://gateway-api.sigs.k8s.io/)
for [ingress (“north-south”) and service mesh (“east-west”) traffic management](/blog/2024/gateway-mesh-ga/).
```

Reply:
https://preliminary.istio.io/latest/docs/releases/feature-stages/ says N-S is Stable and E-W is Beta. If we're going into detail we should be correct; otherwise I'm fine to not go into detail